.github/policies/resourceManagement.yml

```
@@ -224,6 +224,45 @@ configuration:
       - addLabel:
           label: needs-author-feedback
 
+    # /feedbackHub_WAA
+    - description: >-
+          Triggers:
+          * /feedbackhubWCR
+          * /feedbackHubWCR
+      if:
+      - payloadType: Issue_Comment
+      - commentContains:
+          pattern: '\/feedback[H|h]ub'

```

```suggestion
          pattern: '\/feedback[H|h]ubWCR'

```
This should explicitly contain WCR to avoid overlap with the generic feedback hub trigger.